src/CalculatorUI/App.xaml.cs

```diff
@@ -0,0 +1,516 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+//
+// App.xaml.h
+// Declaration of the App class.
+//
+
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using System.Diagnostics;
+using CalculatorApp;
+//using CalculatorApp.Common;
+//using CalculatorApp.Common.Automation;
+using TraceLogging;
+
+//using Microsoft.WRL;
+
+using Windows.ApplicationModel;
+using Windows.ApplicationModel.Activation;
+using Windows.ApplicationModel.Core;
+using Windows.ApplicationModel.Resources;
+using Windows.Foundation;
+using Windows.Foundation.Collections;
+using Windows.Storage;
+using Windows.System;
+using Windows.UI.Core;
+using Windows.UI.Popups;
+using Windows.UI.StartScreen;
+using Windows.UI.ViewManagement;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Controls.Primitives;
+using Windows.UI.Xaml.Data;
+using Windows.UI.Xaml.Input;
+using Windows.UI.Xaml.Interop;
+using Windows.UI.Xaml.Media;
+using Windows.UI.Xaml.Media.Animation;
+using Windows.UI.Xaml.Navigation;
+//using Windows.ApplicationModel.Activation;
+
+namespace CalculatorApp
+{
+    namespace ApplicationResourceKeys
+    {
+        static public partial class Globals
+        {
+            public static readonly string AppMinWindowHeight = "AppMinWindowHeight";
+            public static readonly string AppMinWindowWidth = "AppMinWindowWidth";
+        }
+    }
+
+    /// <summary>
+    /// Provides application-specific behavior to supplement the default Application class.
+    /// </summary>
+    sealed partial class App : Application
```

Comment:
in App.g.h
```
    partial ref class App :  public ::Windows::UI::Xaml::Application,
        public ::Windows::UI::Xaml::Markup::IXamlMetadataProvider
```

Looks like C++/CX's `partial` keyword can be used partially? 🤔

Reply:
Regarding [this article](https://docs.microsoft.com/en-us/cpp/extensions/partial-cpp-component-extensions?view=msvc-160)

> For a ref class that has two partial definitions, the partial keyword is applied to the first occurrence of the definition, and this is typically done by auto-generated code, so that a human coder doesn't use the keyword very often